nengo/tests/test_rc.py

```diff
@@ -0,0 +1,52 @@
+import os
+from nengo import rc
+
+
+def test_read_file():
+    """Tests if files are read properly"""
+
+    file_dir = os.path.abspath(
+        os.path.join(os.path.dirname(__file__), os.pardir, os.pardir)
+    )
+
+    fp = open("%s/nengo/tests/test_rc_file1.txt" % file_dir, "r")
+
+    fp2 = open("%s/nengo/tests/test_rc_file1.txt" % file_dir, "r")
+
+    class TestFile:
```

Comment:
What's the point in testing with this mock file class? This function is just testing `test_read_file`, which is a pretty simple function on our end. I don't think we need this to cover the lines.